index.html

```diff
@@ -4,9 +4,188 @@
+                </div>
+                <div class="main-text">
+                    <span class="primary-head hello">"Hello! Musicians"</span>
+                <h1 class="primary-head">GUITARRIST GLOBAL CONVENTION 2022</h1>
+                <p class="description">The Guitarrist Global Convention(GGC) is a performing and guitar event for players and music-lovers of all ages, levels, and interests!
+                </p>
+                </div>
+                <div class="bottom-text">
+                    <h2 class="landing-date">2022.06.16(THU) - 17(FRI)</h2>
+                    <span class="landing-date place">@ Anaheim Ballroom at Marriott, Anaheim, CA</span>
+                </div>
+            </div>
+        </section>
+
+        <section class="main-program" id="program">
+            <div class="wrapper">
+                <div class="program-container">
+                    <div class="program-title">
+                        <h3>Main Program</h3>
+                        <div class="guide_bar"></div>
+                    </div>
+                    <div class="program-categories">
+                        <div class="program-categories-details">
+                            <img src="images/program_icon_01.png" alt="">
+                            <span>Conferences</span>
+                            <p>Listen to the artists from various countries and bands about the messeges of learning and growing.</p>
+                        </div>
+                        <div class="program-categories-details">
+                            <img src="images/program_icon_02.png" alt="">
+                            <span>Guitars Exhibition</span>
+                            <p>Appreciate various famous guitars, organized from Gibson Guitar Center.</p>
+                        </div>
+                        <div class="program-categories-details">
+                            <img src="images/program_icon_03.png" alt="">
+                            <span>Forum</span>
+                            <p>Have the time to share your thoughts and opinions with music experts.</p>
+                        </div>
+                        <div class="program-categories-details">
+                            <img src="images/program_icon_04.png" alt="">
+                            <span>Workshop</span>
+                            <p>Try creating your own music with your favorite artists and bands</p>
+                        </div>
+                        <div class="program-categories-details">
+                            <img src="images/program_icon_05.png" alt="">
+                            <span>GGC Ignite</span>
+                            <p>Get opportunities to network with GGC affiliates around the world, also after the summit.</p>
+                        </div>
+                        
+                    </div>
+                    <a class="btn-mobile-program">SEE THE WHOLE PROGRAM</a>
```

Comment:
- [ ] Your ''SEE THE WHOLE PROGRAM' link has the same color as the background and does not appear in the desktop version. Please use the colors from the color swatch to make it appear
 -Your design:
<img width="1172" alt="image" src="https://user-images.githubusercontent.com/61919617/190246796-1c18443c-d972-4d3c-a335-7bef127539c3.png">